pkg/machine/ignition.go

```diff
@@ -344,6 +347,40 @@ Delegate=memory pids cpu io
 		},
 	})
 
+	// Setup /etc/subuid
+	files = append(files, File{
+		Node: Node{
+			Group:     getNodeGrp("root"),
+			Path:      "/etc/subuid",
+			User:      getNodeUsr("root"),
+			Overwrite: boolToPtr(true),
+		},
+		FileEmbedded1: FileEmbedded1{
+			Append: nil,
+			Contents: Resource{
+				Source: encodeDataURLPtr(fmt.Sprintf(subUID, usrName)),
+			},
+			Mode: intToPtr(0744),
+		},
+	})
+
+	// Setup /etc/subgid
+	files = append(files, File{
+		Node: Node{
+			Group:     getNodeGrp("root"),
+			Path:      "/etc/subgid",
+			User:      getNodeUsr("root"),
+			Overwrite: boolToPtr(true),
+		},
+		FileEmbedded1: FileEmbedded1{
+			Append: nil,
+			Contents: Resource{
+				Source: encodeDataURLPtr(fmt.Sprintf(subUID, usrName)),
+			},
+			Mode: intToPtr(0744),
+		},
+	})
```

Comment:
Can we remove the duplication by using a for loop, `for _,path := range []string{"/etc/subuid", "/etc/subgid"}`

Reply:
you mean like looping on a slice that has two elements (subuid and subgid) ?

Reply:
yes